packages/block-library/src/navigation/index.php

```diff
@@ -155,8 +155,9 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 
 	if ( empty( $block->inner_blocks ) ) {
 		if ( array_key_exists( '__unstableLocation', $attributes ) ) {
-			$location                 = $attributes['__unstableLocation'];
-			$maybe_classic_navigation = gutenberg_render_menu_from_location( $location, $attributes );
+			$location                         = $attributes['__unstableLocation'];
+			$attributes['__unstableLocation'] = null;
```

Comment:
could this be `unset( $attributes['__unstableLocation'] )`?

Reply:
Yea, that works better, thank you.
I am including [that change in this PR](https://github.com/WordPress/gutenberg/pull/33224/commits/a3cfbfc9039f97eb299d9fd7b4f41696d4cd559f).